site/components/src/main/java/uk/nhs/digital/common/components/EventsComponent.java

```diff
@@ -114,4 +120,20 @@ protected int getPageSize(final HstRequest request, final EssentialsPageable par
         return NumberUtils.isCreatable(compononentPageSize)
             ? Integer.parseInt(compononentPageSize) : super.getPageSize(request, paramInfo);
     }
+
```

Comment:
Do we need to create above methods here?
getComponentParametersInfo
createQueryFilter
applyAndFilters

Reply:
Yes, as the super methods have protected access modifiers and thus are not accessible outside of their class or sub-class